app/helpers/setCookie.js

```diff
@@ -0,0 +1,10 @@
+import { helper } from '@ember/component/helper';
+
+function setCookie(theme) {
+  const domain = '.realdevsquad.com';
+  const cookieExpiryDate = new Date(Date.now() + 24 * 31 * 60 * 60 * 1000);
```

Comment:
Can we please seperate the calculation in a variable? Read more here: https://github.com/ryanmcdermott/clean-code-javascript#use-searchable-names

Reply:
I have bookmarked the repo. Thanks for sharing it.

Reply:
Welcome!

Reply:
Though what I was asking for is
```suggestion
  const COOKIE_DEFAULT_EXPIRY = 31 * 24 * 60 * 60 * 1000; // 31 days
  const cookieExpiryDate = new Date(Date.now() + COOKIE_DEFAULT_EXPIRY);
```

Please note that the sequence of 24 and 31. Earlier I mistook it as 24 days 🤷🏻‍♂️